nodeup/pkg/model/protokube.go

```diff
@@ -101,6 +101,15 @@ func (t *ProtokubeBuilder) Build(c *fi.ModelBuilderContext) error {
 	}
 	c.AddTask(service)
 
+	if t.Cluster.Spec.ContainerRuntime == "containerd" {
```

Comment:
This is not related just to containerd, but in general, because it is needed by Systemd to work properly (at least that is how I remember from testing a while back).

Reply:
At most, the condition should be to install `dbus` if distro is Kopeio.